src/time/src/Time.ts

```diff
@@ -79,7 +93,7 @@ export default defineComponent({
         return mergedFormatRef.value(
           mergedTimeRef.value,
           props.format,
-          dateFnsOptionsRef.value
+          formatOptionsRef.value
```

Comment:
line 103 & line 110 miss changes.

Also, if `dateFnsOptionsRef ` is only referred once after changes applied. We should remove this var and change it to inline code.

Reply:
I used to thought the `formatOptions` should only work when the `format` props is satisfied.
You are right, we should add it to line 103 and 110 to align the options.